utils/service_discovery/consul_config_store.py

```diff
@@ -46,10 +46,13 @@ def get_client(self, reset=False):
 
     def client_read(self, path, **kwargs):
         """Retrieve a value from a consul key."""
-        recurse = kwargs.get('recursive', False)
+        recurse = kwargs.get('recursive', False or kwargs.get('all, False'))
```

Comment:
this looks up the `'all, False'` key. The correct lookup should be:

```py
recurse = kwargs.get('recursive') or kwargs.get('all', False)
```
or
```py
recurse = kwargs.get('recursive', kwargs.get('all', False))
```

depending on whether a "falsey" value of `kwargs['recursive']` should default to a lookup of the `all` key or not